### ISSUE TITLE
Does Flight support namespaced controller?

### ISSUE THREAD

**Issue by ghost** [flightphp/core#227]
Im using Flight because it looks simple & easy to use. But then i need to use namespaced controller. Is this possible with Flight? Also, Flight use statical method of controller. Is it possible to use class with non-statical method?

Thanks :)
Javas

**Comment by tamtamchik** [comment]
@javasworks simple solution - just pass array of object and method name to `route` function:

``` php
Flight::route('route', [new \Namespace\SomeController(), 'method']);
```

**Comment by cos800** [comment]
Why don't you support this way @mikecao 

``` php
Flight::route('/aaa', ['\\my\\aaa','bbb']);
```

and this way

``` php
Flight::route('/bbb', '\\my\\aaa::bbb');
```

**Comment by cos800** [comment]
``` php
    $str = '\\my\\aaa';
    $obj = new $str;
    $obj->bbb(); // It's work.
```

**Comment by cos800** [comment]
``` php
$str = '\\my\\aaa::bbb';
call_user_func_array($str, []); // It's work too.
```

**Comment by mikecao** [comment]
Fixed in commit ec24d786026de89bc4852d6f33de6b42fb0a5b98.
